themes/default/content/blog/aws-static-website-component/index.md

```diff
@@ -0,0 +1,193 @@
+---
+title: Introducing the AWS Static Website Component
+date: 2022-06-27
+meta_desc: Learn how to easily deploy your static website using the AWS Static Website Component. 
+meta_image: "" 
+authors:
+    - sean-holung
```

Comment:
temporarily put me here, since we do not yet have roberta as an author in our system and is causing errors